monkey/common/configuration/agent_sub_configurations.py

```diff
@@ -4,6 +4,24 @@
 
 @dataclass(frozen=True)
 class CustomPBAConfiguration:
+    """
+    Dataclass for the configuration of custom post-breach actions
```

Comment:
```suggestion
    A configuration for custom post-breach actions
```

To some extend it's unavoidable, but let's try to minimize leaking out that it's a dataclass. This is an implementation detail and it could change.